src/components/inputs/ListFilter/FullTextFilterEndAdornment.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import InputAdornment from '@mui/material/InputAdornment'
+import { FullTextFilterEndAdornmentProps } from './types'
+import CloseIcon from '@mui/icons-material/Close'
+import ExpandLessIcon from '@mui/icons-material/ExpandLess'
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
+import FilterListIcon from '@mui/icons-material/FilterList'
+import { IconButton } from 'components'
+
+const FullTextFilterEndAdornment = ({ localizedStrings, resetTextFilter, expandFilters, expanded, openVisibleFiltersMenu, hasChildren, hasVisibleFilters } : FullTextFilterEndAdornmentProps) => {
+  return (
+    <InputAdornment position="end">
+    <IconButton size='small' color='transparent' aria-label='Reset Filters' aria-description={localizedStrings.ResetFilters} tooltip={localizedStrings.ResetFilters} onClick={resetTextFilter}>
+      <CloseIcon />
```

Comment:
avem deja un `type:close` in IconButton.De ce a fost nevoie sa pui tu icon-ul de Close?

Reply:
Eu n-am vazut sa existe `type:close`, am vazut in schimb `type:cancel`, dar nu mi se parea ca arata la fel de bine. In orice caz, am adaugat acum type-uri in IconButton pentru toate icon-urile pe care le-am folosit